readme.md

```diff
@@ -340,31 +338,6 @@ With a token:
 	StripeCard stripeCard = cardService.Create(*customerId*, myCard); // optional isRecipient
 ```
 
-With a card:
-
-```csharp
-	var myCard = new StripeCardCreateOptions();
-
-	// setting up the card
-	myCard.SourceCard = new SourceCard()
-	{
-		Number = "4242424242424242",
-		ExpirationYear = 2022,
-		ExpirationMonth = 10,
-		AddressCountry = "US",                // optional
-		AddressLine1 = "24 Beef Flank St",    // optional
-		AddressLine2 = "Apt 24",              // optional
-		AddressCity = "Biggie Smalls",        // optional
-		AddressState = "NC",                  // optional
-		AddressZip = "27617",                 // optional
-		Name = "Joe Meatballs",               // optional
-		Cvc = "1223"                          // optional
-	};
-
-	var cardService = new StripeCardService();
-	StripeCard stripeCard = cardService.Create(*customerId*, myCard); // optional isRecipient
-```
-
```

Comment:
It's still incorrect to say that. PCI compliance and mobile has implications too if you don't use our official tools and the Stripe.net SDK is not part of our PCI reviewed code so we can't pretend it is.